<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>Recombination and the evolution of base composition in prokaryotic genomes &middot; Samuel Barreto</title>
        <meta name="description" content="Is gene conversion biased toward GC in Bacteria as well ?">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="generator" content="Hugo 0.15" />
        <meta name="robots" content="index,follow">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="stylesheet" href="https://sam217pa.github.io/css/normalize.css">

        

        <link rel="stylesheet" href="https://sam217pa.github.io/css/solarized-light.min.css"> </link>
        <link rel="stylesheet" href="https://sam217pa.github.io/css/style.css">
	<link rel="stylesheet" href="https://sam217pa.github.io/css/override.css">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
        <link href="https://fonts.googleapis.com/css?family=Lato:300|Fira+Mono|Ubuntu:300&subset=cyrillic,cyrillic-ext,latin-ext" rel="stylesheet">
    </head>
    <body>
        
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-69057337-1', 'auto');
ga('send', 'pageview');
</script>


        <div id="wrapper">
            <header class="site-header">
                <div class="container">
                    <div class="site-title-wrapper">
                        
                        <h1 class="site-title">
                            <a title="Bacterial Finches" href="https://sam217pa.github.io/">Bacterial Finches</a>
                        </h1>
                        
                        <a class="button-square" href="https://sam217pa.github.io/index.xml"><i class="fa fa-rss"></i></a>
                        
                        <a class="button-square button-social hint--top" data-hint="Twitter" title="Twitter" href="https://twitter.com/samuel_barreto8">
                            <i class="fa fa-twitter"></i>
                        </a>
                        
                        
                        <a class="button-square button-social hint--top" data-hint="Github" title="Github" href="https://github.com/sam217pa">
                            <i class="fa fa-github-alt"></i>
                        </a>
                        
                        
                        <a class="button-square button-social hint--top" data-hint="Reddit" title="Reddit" href="https://www.reddit.com/user/sam217pa/">
                            <i class="fa fa-reddit-square"></i>
                        </a>
                        
                        
                        
                        
			<h1 class="site-title" id="en">
			    <a title="fr" href="https://sam217pa.github.io/fr/">fr</a>
			</h1>
                    </div>

                    <ul class="site-nav">
                        
    <li class="site-nav-item">
        <a title="Blog" href="/">Blog</a>
    </li>

    <li class="site-nav-item">
        <a title="Projects" href="/project/">Projects</a>
    </li>

    <li class="site-nav-item">
        <a title="Curriculum" href="/page/curriculum/">Curriculum</a>
    </li>

    <li class="site-nav-item">
        <a title="About" href="/page/about/">About</a>
    </li>

                    </ul>
                </div>
            </header>

            <div id="container">


<div class="container">
    <article class="post-container">
        <header class="post-header">
    <h1 class="post-title">Recombination and the evolution of base composition in prokaryotic genomes</h1>
    
        <p class="post-description">Is gene conversion biased toward GC in Bacteria as well ?</p>
    
</header>

        <div class="post-content clearfix">
    

    

<h1 id="what-s-all-the-fuss-about-base-composition:425db19f99baa55def903d554e6fa9b0">What&rsquo;s all the fuss about base composition ?</h1>

<p>Every genome is made of only four nucleotides : Adenine, Thymine, Guanine and
Cytosine. By convention, biologists describe base composition of genomes in term
of GC content, or GC%.</p>

<p>$$
GC\% = \frac{GC}{AT + GC}
$$</p>

<p>Base composition is a fundamental genome trait. It typically varies many fold
over the genome, and between genome. Since fifty years, microbial GC-content
diversity has been described<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:See-for-example"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:See-for-example">1</a></sup>.
Now that we can massively sequence a lot of genome, we know that GC% is even
more variable than expected. Extremely reduced genomes of insect endosymbionts
typically have low GC%. <em>Carsonella rudii</em> is the most AT rich genome sequenced
to date. On the other side of the distribution, large genomes of soil bacteria
have GC content around 70%, like <em>Streptomyces</em>.</p>

<p>The origin of such variability has been the topic of many debates since fifty
years. Some authors have proposed that GC% is under selection, that the GC%
contributes to individual fitness in a given environment. Several lines of
evidence tends to indicate that environmental parameters may be related to GC%,
like oxygen or nitrogen availability, or optimal growth temperature<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:1"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:1">2</a></sup>. But
correlations between such factors and base composition are weak at best, and are
often not robust to analysis in a powerful phylogenetic framework, <em>ie</em> taking
phylogenetic inertia into account. Thus, the favoured hypothesis since 1962 and
the famous work of Sueoka <em>et al</em>, was that divergence in mutational bias
patterns was the main determinants of base composition variability. Sueoka
proposed that GC-rich genomes may have more mutations toward G and C, while
AT-rich genomes may have more mutations toward A and T. This hypothesis have
popularized the idea that base composition is mostly determined by a neutral
process : mutation.</p>

<h1 id="2010-mutational-bias-hypothesis-shattered:425db19f99baa55def903d554e6fa9b0">2010 : Mutational bias hypothesis shattered</h1>

<p>Sueoka&rsquo;s idea was the dominant school of thought during fifty years. This
hypothesis was broken into pieces by one issue of Plos Genetics<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:2"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:2">3</a></sup>, in which two
famous geneticists team independently and simultaneously demonstrated that
mutation is universally biased towards AT, even in GC-rich genomes. They also
showed that some kind of evolutionary force tends to increase GC content in all
genomes. Consequences of this two paper may well represent a &ldquo;seismic shift of
paradigm&rdquo; in microbial evolution, as coined by Rocha &amp; Feil<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:3"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:3">4</a></sup> in their review
paper. It can only mean one thing : some selection or selection-like process is
increasing GC content in bacteria.</p>

<h1 id="why-not-gc-biased-gene-conversion:425db19f99baa55def903d554e6fa9b0">Why not GC-biased gene conversion ?</h1>

<p>In eucaryotes, it has been shown that recombination affects GC content.
Recombination involves pairing of homologous strands of DNA into a structure
called an <em>heteroduplex</em>. Mismatch between strands are corrected by a specific
mechanism, which tends to introduce more GC bases than AT. The process of
mismatch repair leads to gene conversion, <em>ie</em> the unidirectional transfer of
genetic information from the donor strand to the corrected strand. In mammals
and probably a great deal of eucaryotic organisms, gene conversion is biased
towards GC, which means that GC:AT mismatch are more often corrected into GC
than AT. The GC-biased gene conversion (gBGC) consequences have been thoroughly
reviewed by Duret &amp; Galtier in <a href="10.1146/annurev-genom-082908-150001">this</a> 2009
paper. The major consequences is that recombination hotspots (intensely
recombining genomic regions) are GC-richer than coldspots. One can draw a very
strong correlation between GC-content of a genomic region and its
crossover-rate<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:4"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:4">5</a></sup>.</p>

<p>By increasing GC-alleles fixation probability over AT-alleles, gBGC has a
selection-like signature in genomes. When the two 2010 papers came out, my
supervisor thought that maybe gene conversion is also biased toward GC in
bacteria ! Florent Lassalle, during is PhD, thus looked for traces of
recombination in bacterial genomes, using comparative genomic tools<sup class="footnote-ref" id="fnref:425db19f99baa55def903d554e6fa9b0:lassalle"><a rel="footnote" href="#fn:425db19f99baa55def903d554e6fa9b0:lassalle">6</a></sup>. He binned
genomes into twenty distinct GC% bins, and found that the proportion of
recombining genes in a given bin is correlated to its GC%. He also found that
GC% is higher in recombining genes, and even higher at third codon positions.
Due to genetic code redundancy, the third codon position is less strongly
subject to purifying selection.</p>

<blockquote>
<p>All these observations are compatible with the gBGC hypothesis.</p>
</blockquote>

<h1 id="what-s-my-job:425db19f99baa55def903d554e6fa9b0">What&rsquo;s my job ?</h1>

<p>But observations only are what they are : observations. It would be great if we
could experimentally test that gene conversion is biased or not in bacterial
models. My first goal is thus to measure the frequency of conversion of GC:AT
mismatch toward GC and AT. I will force mismatch by using synthetic sequences
that introduces single nucleotide variants at a given locus, and sequence the
recombination products. We need many recombining bacteria. We chose to use
natural transformation in <em>Acinetobacter baylyi</em>, known for its high
transformation frequencies.</p>

<p>My second goal is to estimate fundamental recombination related traits, like
recombination rate per base and global effective population size. We want to
confront those measures with known GC-content affecting environmental traits and
fundamental genome properties, like genome size. The idea is to check for
recombination related confounding patterns in previously advanced
GC%-determining traits.</p>

<p>Maybe we&rsquo;ll then have a more solid overview of what may be at stake in this ever
evading property of bacterial genomesâ€¦</p>

<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>
<div class="footnotes">

<hr />

<ol>
<li id="fn:425db19f99baa55def903d554e6fa9b0:See-for-example">See for example the work of Hill <em>et al</em>, 1966 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:See-for-example"><sup>[return]</sup></a></li>
<li id="fn:425db19f99baa55def903d554e6fa9b0:1">Optimal Growth Temperature was thought to be positively correlated to GC% in the nineties, but this theory has been refuted since. See the work of Galtier <em>et al</em>, 1997. They demonstrated that only structural RNA-encoding DNA GC% is correlated to GC%. There is no significant correlation between GC content and OGT.
 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:1"><sup>[return]</sup></a></li>
<li id="fn:425db19f99baa55def903d554e6fa9b0:2">See <a href="http://dx.doi.org/10.1371/journal.pgen.1001107">this</a> work by Hildebrand <em>et al</em>, and <a href="http://dx.plos.org/10.1371/journal.pgen.1001115">this</a> work by Hershberg &amp; Petrov.
 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:2"><sup>[return]</sup></a></li>
<li id="fn:425db19f99baa55def903d554e6fa9b0:3">See <a href="http://dx.doi.org/10.1371/journal.pgen.1001104">here</a>.
 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:3"><sup>[return]</sup></a></li>
<li id="fn:425db19f99baa55def903d554e6fa9b0:4">See <a href="10.1371/journal.pgen.1000071">this</a> work by Duret &amp; Arndt on the impact of recombination on nucleotide substitutions in the human genome.
 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:4"><sup>[return]</sup></a></li>
<li id="fn:425db19f99baa55def903d554e6fa9b0:lassalle">His <a href="10.1371/journal.pgen.1004941">paper</a> has been published in Plos Genetics in 2015.
 <a class="footnote-return" href="#fnref:425db19f99baa55def903d554e6fa9b0:lassalle"><sup>[return]</sup></a></li>
</ol>
</div>

</div>

        <footer class="post-footer clearfix">
    

    <div class="share">
        <a class="icon-twitter" href="http://twitter.com/share?text=Recombination%20and%20the%20evolution%20of%20base%20composition%20in%20prokaryotic%20genomes&url=https%3a%2f%2fsam217pa.github.io%2fproject%2fprojet-gbc%2f"
            onclick="window.open(this.href, 'twitter-share', 'width=550,height=235');return false;">
            <i class="fa fa-twitter"></i>
            <span class="hidden">Twitter</span>
        </a>

        <a class="icon-facebook" href="https://www.facebook.com/sharer/sharer.php?u=https%3a%2f%2fsam217pa.github.io%2fproject%2fprojet-gbc%2f"
            onclick="window.open(this.href, 'facebook-share','width=580,height=296');return false;">
            <i class="fa fa-facebook"></i>
            <span class="hidden">Facebook</span>
        </a>

        <a class="icon-google-plus" href="https://plus.google.com/share?url=https%3a%2f%2fsam217pa.github.io%2fproject%2fprojet-gbc%2f"
           onclick="window.open(this.href, 'google-plus-share', 'width=490,height=530');return false;">
           <i class="fa fa-google-plus"></i>
            <span class="hidden">Google+</span>
        </a>
    </div>
</footer>
    </article>
</div>

            </div>
        </div>

        <footer class="footer">
            <div class="container">
                <div class="site-title-wrapper">
                    <h1 class="site-title">
                        <a title="Bacterial Finches" href="https://sam217pa.github.io/">Bacterial Finches</a>
                    </h1>
                    <a class="button-square button-jump-top js-jump-top" href="#">
                        <i class="fa fa-angle-up"></i>
                    </a>
                </div>

                <p class="footer-copyright">
                    <span>&copy; 2016 / Powered by <a href="http://gohugo.io/">Hugo</a></span>
                </p>
                
                
                
                
            </div>
        </footer>

        <script src="https://sam217pa.github.io/js/jquery-3.0.0.min.js"></script>
        <script src="https://sam217pa.github.io/js/highlight.pack.js"></script>
        <script>hljs.initHighlightingOnLoad();</script>
        <script src="https://sam217pa.github.io/js/jquery.fitvids.js"></script>
        <script src="https://sam217pa.github.io/js/scripts.js"></script>
    </body>
</html>

